packages/neo-one-smart-contract-compiler/src/compile/syscalls.ts

```diff
@@ -985,7 +995,7 @@ export const SYSCALLS = {
   'Neo.Storage.Find': new SimpleSysCall({
     name: 'Neo.Storage.Find',
     args: [new SysCallArgument('context', StorageContextValue), new SysCallArgument('prefix', StorageKey)],
```

Comment:
This should take a union of `StorageContextValue` and `StorageContextReadonlyValue`